tests/shortcuts.spec.ts

```diff
@@ -23,7 +23,17 @@ test('shortcut downloads logs in full screen error', async ({ page }) => {
 });
 
 test('shortcut downloads logs in embeded-widget', async ({ page }) => {
+  test.slow(); // Extend timeout to prevent a failure before page loads
```

Comment:
Neat, haven't used this API before. Looks like you can limit it based on browser as well, which is nice.
Instead of having a separate comment, you could include it in the description of the `test.slow`, e.g.
```suggestion
  test.slow(true, "Extend timeout to prevent a failure before page loads");
```